earthly/python/pyproject.toml

```diff
@@ -0,0 +1,14 @@
+[tool.poetry]
```

Comment:
same

Reply:
Deleted the files in commit 4f22d7aea18d573a03bc3ca0b87efbc43add4da0